### ISSUE TITLE
HTTP.createServer then close after one connection

### ISSUE THREAD

**Issue by jamesdbrock** [purescript-node/purescript-node-http#46]
I want to `close` an `HTTP` server after one connection. It seems to be impossible? The variable `server` is not defined at `close server`.

```
  server <- createServer \request response -> do
    respond request response
    close server (pure unit)
```

It looks like this is impossible because the *Node.js* API is bad. `createServer` is just a wrapper for the *Node.js* `http.createServer` function. But still.

**Comment by jamesdbrock** [comment]
Ok I see, we're missing the `onRequest` event. That's what we need.

**Comment by JordanMartinez** [comment]
Fixed by #49.
